code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2329,3 +2329,38 @@
 		to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "The feeling of a hairball...", "Backrubs would be nice.", "Whats behind those doors?")]</span>")
 	M.adjustArousalLoss(2)
 	..()
+
+/datum/reagent/berserk
+	name = "hell powder"
+	id = "berserk"
+	description = "A drug from Hell that increases the user's muscle strength to great levels."
+	color = "#751717"
+	taste_description = "rage"
+	overdose_threshold = 50
```

Comment:
suggestion made their PH be like 12~ so people dont meme them into beakers to get 50u injectors?

Reply:
that would actually be funny emergent gameplay, bluespace syringe someone and they can't hold items